workflows/tf-object-detection-training/utils.py

```diff
@@ -0,0 +1,185 @@
+
+import os
+import json
+import yaml
+import csv
+import numpy as np
+import tensorflow as tf
+from google.protobuf import text_format
+from object_detection.protos import pipeline_pb2
+
+def get_default_params(model):
+    with open('defaults.json', 'r') as f:
+        default_dict = json.load(f)
+    return default_dict[model].copy()
+
+def legacy_params_compatibility(params):
+    new_params = params.copy()
+    if "num-clones" in params:
+        new_params["num_clones"]=params["num-clones"]
```

Comment:
can you add spaces before and after `=`?

Reply:
Done.